src/Symfony/Component/DependencyInjection/Tests/Loader/XmlFileLoaderTest.php

```diff
@@ -486,4 +486,14 @@ public function testAutowire()
 
         $this->assertTrue($container->getDefinition('bar')->isAutowired());
     }
+
+    /**
+     * @group legacy
+     */
+    public function testAliasDefinitionContainsUnsupportedElements()
+    {
+        $container = new ContainerBuilder();
+        $loader = new XmlFileLoader($container, new FileLocator(self::$fixturesPath.'/xml'));
+        $loader->load('legacy_invalid_alias_definition.xml');
```

Comment:
Why not adding a test that the deprecation is actually triggered (we did something similar in the Yaml component)?